cypress/e2e/logout.cy.js

```diff
@@ -0,0 +1,15 @@
+describe('Logout Test', () => {
+  const validEmail = 'test123@stud.noroff.no';
+  const validPassword = 'test1234';
+
+  it('enables a logged-in user to log out successfully', () => {
+    cy.visit('/');
+    cy.get('#registerModal').contains('Login').click();
+    cy.get('#loginForm').should('be.visible');
+    cy.get('#loginEmail').type(validEmail);
+    cy.get('#loginPassword').type(validPassword);
+    cy.get('button[type=submit]').contains('Login').click();
+
+    cy.get("button[data-auth='logout']").click();
```

Comment:
Same here, you need to check localStorage for the token

Reply:
Hi. I have now made the changes requested :-)